### PR TITLE
feat: Add zodiac signs to the day & month view + refactor json data loader

### PR DESCRIPTION
**Pull request by zohary89**
- Add the zodiac signs with start and end day and month to the db
- Add images of the 12 zodiac signs (images from wikipedia- no copyright)
- The sign will appear in the day view. (with an existing  function for the option to add into the month view in the future. Each sign will contain a link to an external website with info about the sign.
- create generic json data loader (change the load of the daily quotes and convert it into generic module with relavent tests changes)- everyone can use this module for loading required data from json structure..
- In the future- option to add a setting in the profile settings panel - whether to display the zodiac signs or not.

### CODE REVIEW

File: app/internal/json_data_loader.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+def get_data_from_json(path: str) -> List[Dict[str, Union[str, int, None]]]:
+    """This function reads all of the data from a specific JSON file.
+    The json file consists of list of dictionaries"""
+    try:
+        with open(path, 'r') as f:
+            json_content_list = json.load(f)
+    except (IOError, ValueError):
+        return []
```
Comment (yammesicka) [inline app/internal/json_data_loader.py:18]: Maybe we should log some warning? :)

File: app/internal/json_data_loader.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import json
```
Comment (yammesicka) [inline app/internal/json_data_loader.py:1]: Cool, I really love the way you've refactored it

File: app/internal/json_data_loader.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+def is_table_empty(session: Session, table: Base) -> bool:
+    return session.query(table).count() == 0
+
+
+def load_data(
+        session: Session, path: str,
+        table: Base, object_creator_function: Callable) -> None:
+    """This function loads the specific data to the db,
+    if it wasn't already loaded"""
+    if is_table_empty(session, table):
```
Comment (yammesicka) [inline app/internal/json_data_loader.py:31]: Prefer guard close to return early: if not is_table_empty return None.

File: app/internal/zodiac.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        end_month=zodiac_fields['end_month'],
+        end_day_in_month=zodiac_fields['end_day_in_month']
+        )
+
+
+def get_zodiac_of_day(session: Session, date: date) -> Zodiac:
+    """This function return a zodiac object
+    according to the current day."""
+    zodiac_obj = session.query(Zodiac).filter(
+        or_((and_(Zodiac.start_month == date.month,
```
Comment (yammesicka) [inline app/internal/zodiac.py:26]: This one is a bit tricky. Use `_between` or export some expression to a variable with an indicative name :)

File: tests/zodiac_fixture.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import pytest
+from sqlalchemy.orm import Session
+
+from app.database.models import Zodiac
+from tests.utils import create_model, delete_instance
+
+
+@pytest.fixture
```
Comment (yammesicka) [inline tests/zodiac_fixture.py:8]: Awesome job

**Review by yammesicka — CHANGES_REQUESTED** [review]
High quality work! Great job. I do see how some minor improvement can make it perfect, please take a look :)

File: app/internal/quotes/load_quotes.py [hunk 6]
```diff
@@ -1,41 +0,0 @@
-import json
```
Comment (yammesicka) [inline app/internal/quotes/load_quotes.py:1]: You should remove this file using `git rm`

File: app/internal/json_data_loader.py [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+def get_data_from_json(path: str) -> List[Dict[str, Union[str, int, None]]]:
+    """This function reads all of the data from a specific JSON file.
+    The json file consists of list of dictionaries"""
+    try:
+        with open(path, 'r') as f:
+            json_content_list = json.load(f)
+    except (IOError, ValueError) as err:
+        file_name = os.path.basename(path)
+        logger.warning(
```
Comment (yammesicka) [inline app/internal/json_data_loader.py:21]: Prefer `logger.exception`

File: tests/test_load_quotes.py [hunk 8]
```diff
@@ -1,25 +0,0 @@
-from app.database.models import Quote
```
Comment (yammesicka) [inline tests/test_load_quotes.py:1]: Remove using `git rm`

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/internal/quotes/load_quotes.py (hunk 6 above)
Comment (zohary89) [reply app/internal/quotes/load_quotes.py:1]: I think I've already removed these files.. I did git rm before the first commit and now again after the pull upstream. I dont have these files in my updated branch...
Maybe I missed something?

File: app/static/images/zodiac/Taurus.svg [hunk 9]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (yammesicka) [inline app/static/images/zodiac/Taurus.svg:1]: Remove Thumbs.db :)

**Review by yammesicka — COMMENTED** [review]
Last push ;)

File: app/internal/quotes/load_quotes.py (hunk 6 above)
Comment (yammesicka) [reply app/internal/quotes/load_quotes.py:1]: Maybe `git rm --cached [full_file_path]`.

Remember to `git commit` and `git push` after running `git rm`.

Comment (zohary89) [reply app/internal/quotes/load_quotes.py:1]: Yes but I did it.. I don't understand, the deleted files are no longer in my branch and in the PR it is written that they are deleted..

Comment (zohary89) [reply app/internal/quotes/load_quotes.py:1]: In addition to my previous comment, I want to make sure we are aligned - both of the deleted files exist in the develop branch(already merged). I replaced them as I refactored the code. Maybe that's why they appear in the pr.
And as I understand, they will be removed just after the merge back to develop...

**Pull request merged**